akka-docs-dev/rst/java/stream-introduction.rst

```diff
@@ -45,6 +45,21 @@ implementations like Akka Streams offer a nice user API).
 
 .. _Reactive Streams: http://reactive-streams.org/
 
+Relationship with Reactive Streams
+----------------------------------
+
+The Akka Streams API is completely decoupled from the Reactive Streams
+interfaces. While Akka Streams focus on the formulation of transformations on
+data streams the scope of Reactive Streams is just to define how to move data
+across an asynchronous boundary without losses, buffering or resource
```

Comment:
while that is true, the point is interoperability, to have a standard interface for such boundaries.